setup.py

```diff
@@ -9,7 +9,7 @@
     ])
 
 install_requires = [
-    'Django >= 1.6, < 1.7',
+    'Django >= 1.9, < 1.10',
```

Comment:
Is er een specifieke reden waarom 1.10 niet kan (over anderhalve maand ofzo)? Indien er echt een specifieke reden is: die er even als commentaar bij zetten.